### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by pypingou**
Relates to https://github.com/ralphbean/pep8bot/issues/19

### CODE REVIEW

**Comment by pypingou** [comment]
Untested due to #26

**Comment by ralphbean** [comment]
Pretty cool.  :)

File: pep8bot/models/__init__.py [hunk 1]
```diff
@@ -177,7 +177,8 @@ class Repo(Base):
     description = Column(Unicode, nullable=False)
     language = Column(Unicode, nullable=False)
     username = Column(Unicode, ForeignKey('users.username'))
-    commits = relation('Commit', backref=('repo'))
+    commits = relation(
+        'Commit', backref=('repo'), order_by='Commit.created.desc()')
```
Comment (ralphbean) [inline pep8bot/models/__init__.py:188]: what does this do?  Does it alter the table?  do we need an alembic upgrade script for this?

File: pep8bot/views/__init__.py [hunk 2]
```diff
@@ -129,11 +129,33 @@ def webhook(request):
     return "OK"
 
 
+@view_config(context=m.Repo)
```
Comment (ralphbean) [inline pep8bot/views/__init__.py:132]: This needs testing now that #26 is merged.

File: pep8bot/models/__init__.py (hunk 1 above)
Comment (pypingou) [inline pep8bot/models/__init__.py:188]: It's purely SQLAlchemy related, it orders the commits accessed by `obj.commits` via their field `created` and in descendant order.

This means that `obj.commits[0]` should refer to the most recent commit by date.

File: pep8bot/views/__init__.py (hunk 2 above)
Comment (pypingou) [inline pep8bot/views/__init__.py:132]: That's for sure :)

**Comment by pypingou** [comment]
With the latest changes, I got it to work but apparently not the worker, so I can't really test it, but afaics, it's not yet working :)

**Comment by ralphbean** [comment]
This doesn't work and isn't merge-able.  Let's close it for now.  If it gets cleaned up, of course we could consider it for merge again.

**Pull request closed without merging**